Core/NoOpRunner.Core/PlayerStates/IdleState.cs

```diff
@@ -0,0 +1,32 @@
+﻿using NoOpRunner.Core.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NoOpRunner.Core.PlayerStates
+{
+    class IdleState : PlayerState
+    {
+        public override Uri doAnimation()
```

Comment:
PascalCase method names, here and in other files as well. Also, `doAnimation` does not sound right, maybe `GetAnimationUri` or smth?

Reply:
Fixed